src/main/java/grabber/formats/EPUB.java

```diff
@@ -30,27 +34,41 @@
             "</head>" + NL +
             "<body>" + NL;
     static final String htmlFoot = "</body>" + NL + "</html>";
-    private final Novel novel;
+    private Novel novel;
     private final NovelMetadata novelMetadata;
-    private final Book book;
+    private Book book;
 
     public EPUB(Novel novel) {
         this.novel = novel;
         this.novelMetadata = novel.metadata;
-        book = new Book();
-        try {
-            book.getResources().add(new Resource(getClass().getResourceAsStream("/default.css"), "default.css"));
-        } catch (IOException e) {
-            GrabberUtils.err(novel.window, "Could not add default.css file to EPUB. "+e.getMessage(), e);
+        // Library novels try to update existing files
+        if (novel.window.equals("checker")) {
+            try {
+                book = readOldFile();
+            } catch (IOException e) {
+                GrabberUtils.err("Could not read old book file.");
+                GrabberUtils.info("Creating new file.");
+            }
+        }
+        if (book == null) {
+            book = new Book();
+            try {
+                book.getResources().add(new Resource(getClass().getResourceAsStream("/default.css"), "default.css"));

```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [UI: Usage of GetResource may be unsafe if class is extended (UI_INHERITANCE_UNSAFE_GETRESOURCE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#ui-usage-of-getresource-may-be-unsafe-if-class-is-extended-ui-inheritance-unsafe-getresource).
Calling this.getClass().getResource(...) could give results other than expected if this class is extended by a class in another package.